python/README.md

```diff
@@ -1,20 +1,44 @@
-## nGraph Neural Network compiler
-
-[nGraph][ngraph_github] is Intel's open-source graph compiler and runtime for Neural Network models. Frameworks using nGraph to execute workloads have shown up to [45X](https://ai.intel.com/ngraph-compiler-stack-beta-release/) performance boost compared to native implementations.
-
-nGraph can be used directly thought it's [Python API][api_python] or [C++ API][api_cpp]. Alternatively it can be used through one of its frontends, e.g. [TensorFlow][frontend_tf], [MXNet][frontend_mxnet] and [ONNX][frontend_onnx].
+## nGraph Compiler stack
+
+[nGraph][ngraph_github] is an open-source graph compiler for Artificial 
+Neural Networks (ANNs). The nGraph Compiler stack provides an inherently 
```

Comment:
`inherently` - not sure if that word adds anything meaningful and can be confusing.

Reply:
Inherently means it already contains or documents or helps with compiling all kinds of workloads across many different kinds of backends without having to install a bunch of different things.  Like Anaconda requires a pretty heavy install for all the different things it needs.